en/config/actuators.md

```diff
@@ -1,94 +1,255 @@
+
+A quadcopter might have an setup screen similar to the one shown below.
+This defines a 4 motor copter with X-geometry, and maps each of the motors to DShot ESC.
+It also maps PWM400 AUX outputs for controlling a parachute and landing gear.
 
 ![Actuators MC (QGC)](../../assets/config/actuators/qgc_actuators_mc_aux.png)
 
-Note that some settings are hidden unless you select the **Advanced** checkbox in the top right corner.
+Note that only the most common settings are displayed by default.
+Select the **Advanced** checkbox in the top right corner to display all settings.
+
 
 ## Geometry
 
-The geometry section is used to configure any additional geometry-related settings for the selected [airframe](../config/airframe.md).
-The UI displays a customised view for the selected type; if there are no configurable options this may display a static image of the frame geometry, or nothing at all.
+The geometry section is used to set any configurable geometry-related parameters for the selected [airframe](../config/airframe.md).
+This includes the number and position of [motors](#motor-geometry), and the number, function, and properties of [control surfaces](#control-surfaces-geometry).
+For VTOL tiltrotor vehicles, it will also include the number and properties of [tilt servos](#motor-tilt-servo-geometry)
+
+The UI is customised for the selected airframe.
+Only motors and/or control surfaces that are relevant to the airframe type are displayed or available to assign.
 
-The screenshot below shows the geometry screen for a multicopter frame, which allows you to select the number of motors, their relative positions on the frame, and their expected spin directions (select "**Direction CCW**" for counter-clockwise motors).
-This particular frame also includes an image showing the motor positions, which dynamically updates as the motors settings are changed.
+:::note
+Not all aspects of a geometry are necessarily configurable through this interface.
+Factors that cannot be configured are hard coded for the current airframe. 
+:::
 
-![Geometry MC (QGC)](../../assets/config/actuators/qgc_actuators_mc_geometry.png)
+### Motor Geometry
 
-A fixed wing airframe would instead display the parameters that define control surfaces, while a VTOL airframe could display both motors and control surfaces.
+The motor geometry section lets you set the number of motors, the relative position, and other properties of each motor. The motor properties are similar for all frames, but there are some exceptions.
 
+Multicopter airframe provides a diagram in QGC showing the X, Y positions, which is useful for checking whether the motors configuration is correct. To get a feeling for the motor layout for VTOL and other vehicles see the [Airframe Reference](../airframes/airframe_reference.md)
 
-### Conventions
+The motor geometry explanations for a number of different frames are described below.
```

Comment:
FYI This says "you are describing the explanations" not "describing the frames". I will fix.